src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -1662,6 +1662,23 @@ def is_creation_allowed_for_organization(self, organization: Organization) -> bo
     def is_creation_allowed_for_project(self, project: Project) -> bool:
         return True  # Detection always allowed by project for now
 
+    def is_event_eligible(cls, event):
+        tags = event.get("tags", [])
+        browser_name = next(
+            (tag[1] for tag in tags if tag[0] == "browser.name" and len(tag) == 2), None
+        )
+        if browser_name in [
+            "Chrome",
+            "Firefox",
+            "Safari",
+            "Edge",
+        ]:
```

Comment:
Assuming that these browser names always begin with a capital, no need to use .toLower(), etc

Reply:
I think -yes- but I've been wrong about the event data before, so I'm going to call it just in case.